tests/integration/api_build_test.py

```diff
@@ -201,6 +201,27 @@ def test_build_isolation(self):
         for _chunk in stream:
             pass
 
+    @requires_api_version('1.38')
+    def test_build_with_buildkit(self):
+        script = io.BytesIO('\n'.join([
+            'FROM scratch',
+            'COPY <<EOF greeting.txt',
+            'hello world',
+            'EOF'
```

Comment:
The test should use BuildKit-specific features such as `RUN --mount=type=secret`

Reply:
It is using Buildkit-specific features: `COPY` with here documents. Try setting version=1 and you'll watch the test fail.

Reply:
Sorry, I overlooked the BuildKit-specific heredoc. Could you add a code comment to clarify the intent?